### DISCUSSION TITLE
[Bug] Game freezes after startup.

### DISCUSSION THREAD

**Discussion by dragonwing11** [KnuckleCracker/CW4-bug-tracker#1076] [General]
I´ve encountered this every time i start the game. 
The game loads, music starts and the cursor is displayed. everything else freezes. either in the initial load screen, any menu or a few milliseconds into a mission.

The only time i was able to play the game was, when i updated my nvidia drivers on the same session. On system restart, the game crash happens again.

[Player.log](https://github.com/KnuckleCracker/CW4-bug-tracker/files/10158874/Player.log)

**Comment by Karsten75** [comment]
The issue seems to be around your graphics card. The entire log file is filled with DirectX 3d errors:

```
StoreStats - success
d3d11: failed to create buffer (target 0x1 mode 1 size 162960) [0x887A0005]
d3d11: failed to create buffer (target 0x2 mode 1 size 6756) [0x887A0005]
d3d11: attempt to lock null buffer
d3d11: attempt to lock null buffer
d3d11: attempt to lock null buffer
d3d11: attempt to lock null buffer
d3d11: attempt to lock null buffer
```
and so on to the end of the file. 

Preceding this, there doesn't seem to be any other issues at all, so it is quite baffling. 

You may want to attempt a complete uninstall and re-install of your NVIDIA drivers. 

Additionally, you may want to try and run the DirectX diagnostic tool 
https://www.intel.com/content/www/us/en/support/articles/000022556/graphics.html

Other than that, I have no idea.

**Comment by knucracker** [comment]
Moved to a discussion.  Yeah, run dxdiag and have a look. It is built into windows 10/11.  Just type "dxdiag" in your search bar and it should come up.  The error you see is definitely DirectX or driver related and isn't something generally seen.
